shared/provision/code-page/index.tsx

```diff
@@ -499,9 +520,17 @@ const styles = Styles.styleSheetCreate(
         },
       }),
       deviceIcon: {
+        height: 32,
+        width: 32,
+      },
+      deviceIconDesktop: {
         marginLeft: Styles.globalMargins.xtiny,
         marginRight: Styles.globalMargins.xxtiny,
       },
+      deviceIconMobile: {
```

Comment:
seems offset to the right? 
<img width="145" alt="image" src="https://user-images.githubusercontent.com/11968340/76464001-dbbec180-63ba-11ea-887a-9d27e70cf512.png">

Reply:
Mobile, post fix:
![image](https://user-images.githubusercontent.com/917230/76527563-bf309100-646f-11ea-9dad-a019fa50052b.png)
![image](https://user-images.githubusercontent.com/917230/76527658-e4bd9a80-646f-11ea-93dd-7767318270a4.png)

Reply:
![image](https://user-images.githubusercontent.com/917230/76528165-a96f9b80-6470-11ea-9415-36a7b50ee248.png)
![image](https://user-images.githubusercontent.com/917230/76528174-ac6a8c00-6470-11ea-855d-727033389aa4.png)

Reply:
Looks good